Dockerfile

```diff
@@ -2,8 +2,13 @@ FROM python:3.8
 
 RUN apt-get update && apt-get install -y ffmpeg
 RUN pip install poetry==1.0.5
-COPY . /tmp/meme-bot
-COPY fonts/* /usr/share/fonts/truetype/
 WORKDIR /tmp/meme-bot
-RUN poetry install --no-dev
-CMD [ "poetry", "run", "bot" ]
+COPY poetry.lock pyproject.toml ./
+RUN poetry install --no-dev --no-root
+COPY fonts/* /usr/share/fonts/truetype/
+COPY assets ./assets
+COPY meme_generator ./meme_generator
+COPY memebot ./memebot
```

Comment:
Is there a reason you don't just copy the entire directory? Curious, not particularly convinced one way or the other is right

Reply:
just cuz it wasn't necessary? seems like docker wants you to cache as much as possible. so if something changes in the `README.md` for example, there really isn't a need to do the next `RUN poetry install --no-dev --no-interaction`.

that said, looks like i missed tests there =P. so i'll add those